docs/src/testing/index.md

```diff
@@ -0,0 +1,5 @@
+# `fuels-rs` Testing
+
+This section gives you some pointers for testing in `fuels-rs`.
+
+- [`run_script`](./run-script.md)
```

Comment:
This link doesn't work :'( . I think you can directly put the content of `run_script.md` in here, without a link, because the section is relatively empty right now.

Reply:
I added it because i thought we might write more documentation about testing.